API-GUIDE.md

```diff
@@ -2,99 +2,246 @@
 
 ## Access the API
 
+### Users
+
+#### Get Current User
+
+```php
+$result = $client->getCurrentUser();
+```
+
+API Signature:
+```php
+'getCurrentUser' => ['get', 'members/me']
```

Comment:
I love the idea of including the http method and actual trello api signature. I think including this odd snippet from the ApiMethodsTrait maybe confusing as PHP code. It sort of looks like you need to pass that array as arguments or something. Can we update this format to be plaintext like....

`GET /members/me`
`GET /actions/{actionId}`

... something a bit less technical and still conveying the important information.

Reply:
I see. I was quoting directly from the PHP code, hence the method used.
I'll change that.